### PR TITLE
Okta to Auth0 migration - Round 1 Components

### PR DESCRIPTION
**Pull request by SethMacP**
### Motivation for the feature:
 - We are migrating all the listed components from using Okta to Auth0. 
 1. DrawingSubContainer + Tests
 2. WeeklySubmissions
 3. EditPlayersContainer
 4. HelpContainer + Tests
 5. GalleryContainer
 6. FamilySettingsContainer + Tests

### What changes were made?
 - Replaced all instances of {useOktaAuth} hook with the now imported hook, {useAuth0}.
 - Replaced {authState} with {user}, per Auth0 documentation
 - Removed extra code that was required with Okta, but no longer necessary with Auth0
 - Tests updated to mock Auth0

### What issues were fixed?
 - Okta will no longer be used and Auth0 will be the replacement.
 - This is part of a larger, ongoing task to migrate all components away from Okta

### Outside Reference:
[Trello Card for task](https://trello.com/c/43KZzx61)

## Commits Checklist
- [x] Commits have clear title names 
- [x] Commit has descriptive message of what has changed 
- [x] More small commits than less big commits 

## Code Checklist 
- [x] Code is formatted appropriately 
- [x] Code meets DRY standards
- [x] No dead code (check for logs and print statements)  
- [x] No TODOS instead necessary comments 
- [x] Code follows Story Squad standards for:
     - Language 
     - Framework 
     - Libraries

### CODE REVIEW

File: src/components/pages/Help/HelpContainer.js [hunk 1]
```diff
@@ -1,41 +1,18 @@
...
-import { useOktaAuth } from '@okta/okta-react';
+import React, { useState } from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
 
 import Help from './Help';
 
 function HelpContainer({ LoadingComponent }) {
-  const { authState, authService } = useOktaAuth();
+  const { user, isAuthenticated } = useAuth0();
   const [userInfo, setUserInfo] = useState(null);
```
Comment (kdneldor) [inline src/components/pages/Help/HelpContainer.js]: Hey Seth! I just noticed here on line 8 that the useState is set to null. In the other files we have it initialized with "user".

Comment (SethMacP) [reply src/components/pages/Help/HelpContainer.js]: Good catch! I thought I had them all, but this one was missed. I'll push these changes now.

**Review by kdneldor — APPROVED** [review]
Looks good to me! We'll coordinate this with the rest of the main PR's as we merge. Praise the sun!

File: src/components/pages/Help/HelpContainer.js (hunk 1 above)
Comment (kdneldor) [reply src/components/pages/Help/HelpContainer.js]: Thanks! It does make me want to go back and look at all of mine now. I certainly could have missed the same thing.

**Review by kwnie — APPROVED** [review]
Good work!

**Comment by da-vazquez** [comment]
Great Job Seth! Very clean organized code and commits

**Review by allraec — APPROVED** [review]
Good job! Clean and organized code with very clear commits. Easy to navigate through the changes.

**Pull request merged**